apps/web/src/components/Main.tsx

```diff
@@ -388,7 +388,7 @@ export default function Main({ sidebarOpen }: { sidebarOpen: boolean }) {
                 autoFocus: true,
                 onChange: (e) => setValue(e.target.value),
                 onKeyDown: (e) => {
-                  if (e.key === "Enter" && (e.ctrlKey || e.metaKey)) {
+                  if (e.key === "Enter" && !e.shiftKey) {
                     onSend();
```

Comment:
can you please add a `e.preventDefault()` here? 
Hitting enter will also create a newline (and run the onsend() fn)

preventDefault prevents this